### ISSUE TITLE
Demarkation of application .desktop files created

### ISSUE THREAD

**Issue by Aman9das** [A6GibKm/silverblue-tools#6]
I suggest the .desktop files created be placed in a subdirectory of `~/.local/share/applications` such as `./toolbox/` for easier organization. Would that be viable?

**Comment by A6GibKm** [comment]
Yes, but I have no idea if those are read by the DE just the same as desktop files in the applications folder

**Comment by Aman9das** [comment]
Here are the specifications: https://specifications.freedesktop.org/menu-spec/latest/ar01s02.html

It writes about using each directory in the search path (vague), but checking in GNOME by moving `.desktop` files around, only directories in `$XDG_DATA_DIRS/applications/` are valid.
